lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+[
```

Comment:
Should this be duplicated across languages instead of having all translated names in one file to match the structure of other resources?

Reply:
i thought about this, but that requires a whole lot of duplicated keys with redundant info..... not sure whats best?

Reply:
I think separate files are easier for people to extend and less prone to typo errors in keys (`official_name_xx`). Maybe a common resource for country code, native lang name?